app/src/main/java/com/nsc/covidscore/LocationSettingsPageFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.nsc.covidscore;
```

Comment:
Is this file being used? If it's part of a future PR, just remove it from this one so that develop has minimal unused code :)

Reply:
Looks like it's from develop branch. It does appear to be getting used; maybe for something related to the Google Play Store? Is this familiar to you, @dansjack ?

Reply:
It's not familiar to me, there's no `LocationSettingsPageFragment` on develop. It was added to your branch in this commit on October 26th a20c36d3a7a2dcf515e912a21ce4100cdd3bb4ca

Reply:
It looks like an auto-generated file. When you create fragments like this, it creates a layout and java file, so that could be where it came from
<img width="500" alt="image" src="https://user-images.githubusercontent.com/30481844/99466654-a5462880-28f1-11eb-8638-f203726f4c56.png">

Reply:
Oh sorry everyone, I thought Larissa was referring to the literal highlighted line `package com.nsc.covidscore;` not the whole file.

The currently unused frag has been deleted.